core/game.py

```diff
@@ -148,10 +145,36 @@ def get_remaining_poll_targets(self, user: User) -> List[str]:
             if self.user_data[user][person] is None
         ]
 
-    def get_answers_about(self, user: User) -> Dict[str, List[Answer]]:
+    def get_answers_about(self, user: User):
         return {u.name: self.user_data[u][user]
                 for u in self.user_data if (u != user and self.user_data[u][user] is not None)}
 
+    def get_all_answers(self) -> PollResults:
+        question_to_answers_mapping = {}
+        for user in self.user_data:
+            for user1 in self.user_data[user]:
+                for answer in self.user_data[user][user1]:
+                    if answer.question_name not in question_to_answers_mapping:
+                        question_to_answers_mapping[answer.question_name] = {user.name: {user1.name: answer.answer}}
+                    elif user.name not in question_to_answers_mapping[answer.question_name]:
+                        question_to_answers_mapping[answer.question_name][user.name] = {user1.name: answer.answer}
+                    else:
+                        question_to_answers_mapping[answer.question_name][user.name][user1.name] = answer.answer
+        return PollResults(results=[SinglePersonPollResults(personName=u.name,
+                                        questions=[
+                                            SingleQuestionPollResults(
+                                                question=q,
+                                                answers=[
+                                                    SingleAnswer(
+                                                        respondentName=v.name,
+                                                        answer=question_to_answers_mapping[q][v.name][u.name]
+                                                    )
+                                                    for v in self.user_data[u]
+                                                ]
+                                            )
+                                            for q in question_to_answers_mapping]) for u in self.user_data])
+        # return [FilledPoll(answers=self.user_data[u][v], user_about=v, user_filling=u) ]
+
```

Comment:
Ciężkie do zweryfikowania poprawności, a nie ma testu
Jak to przetestowałeś na czymś no to ok

Reply:
testowałem swaggerem + działa na froncie